app/views/courses/TeacherClassesView.coffee

```diff
@@ -0,0 +1,127 @@
+RootView = require 'views/core/RootView'
+template = require 'templates/courses/teacher-classes-view'
+Classroom = require 'models/Classroom'
+Classrooms = require 'collections/Classrooms'
+Courses = require 'collections/Courses'
+Campaign = require 'models/Campaign'
+Campaigns = require 'collections/Campaigns'
+LevelSessions = require 'collections/LevelSessions'
+CourseInstance = require 'models/CourseInstance'
+CourseInstances = require 'collections/CourseInstances'
+ClassroomSettingsModal = require 'views/courses/ClassroomSettingsModal'
+InviteToClassroomModal = require 'views/courses/InviteToClassroomModal'
+User = require 'models/User'
+utils = require 'core/utils'
+helper = require 'lib/coursesHelper'
+
+module.exports = class TeacherClassesView extends RootView
+  id: 'teacher-classes-view'
+  template: template
+  
+  events:
+    'click .edit-classroom': 'onClickEditClassroom'
+    'click .archive-classroom': 'onClickArchiveClassroom'
+    'click .unarchive-classroom': 'onClickUnarchiveClassroom'
+    'click .add-students-button': 'onClickAddStudents'
+    'click .create-classroom-button': 'onClickCreateClassroom'
```

Comment:
Changed. But it looks like "-button" (701 instances) is a lot more common in our codebase than "-btn" (215 instances).